samples/TR/index.html

```diff
@@ -87,13 +87,8 @@
 	The first one (“Classic”) shows it in its usual form.
 	The second one (“Vivliostyle”) shows the same document
 	in a paginated yet responsive view
+	(spread view if window is wide)
```

Comment:
If we wait for `spread=auto`, no problem. If you want to get the other changes from this pull request published first, maybe this should be split into several pull requests.